### ISSUE TITLE
Move interactive element handlers into a child of the container

### ISSUE THREAD

**Issue by gpbl** [gpbl/react-day-picker#392]
This code in `DayPicker.js`:

```jsx
<div
  {...this.props.containerProps}
  ref={el => (this.dayPicker = el)}
  role="application"
  lang={this.props.locale}
  className={className}
  tabIndex={this.props.canChangeMonth && this.props.tabIndex}
  onKeyDown={this.handleKeyDown}
  onFocus={this.props.onFocus}
  onBlur={this.props.onBlur}
>
    {this.renderNavbar()}
    {this.renderMonths()}
</div>
```

is causing the eslint rule [jsx-a11y/no-noninteractive-element-interactions](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md) to fail because the `<div>` has a `role="application"` with element handlers in it. The correct HTML would be instead:

```jsx
<div
  {...this.props.containerProps}
  className={className}
  ref={el => (this.dayPicker = el)}
  role="application"
  lang={this.props.locale}
>
  <div
    tabIndex={this.props.canChangeMonth && this.props.tabIndex}
    onKeyDown={this.handleKeyDown}
    onFocus={this.props.onFocus}
    onBlur={this.props.onBlur}
  >
    {this.renderNavbar()}
    {this.renderMonths()}
  </div>
</div>
```

This change will break the current CSS (and the css class names passed to`classNames`).

- [x] move the element handlers into a child div
- [x] update tests to use the child for handling events
- [x] add a new css class (e.g. `DayPicker-wrapper`) to the `classNames`
- [x] update style.css to replicate the current design
